### PR TITLE
Ensure we stop droning, in the event our free fight familiar is gooso…

### PR DESCRIPTION
**Pull request by Ignose**
… to charge the meat weight

### CODE REVIEW

File: packages/garbo/src/lib.ts [hunk 1]
```diff
@@ -1146,7 +1148,8 @@ export function scalerCap(monster: Monster): number {
   return cap;
 }
 
-const STRONG_SCALER_THRESHOLD = 1_000;
+// What constitutes a strong scaler depends on our ability to take hits. Let's compare mainstat with 50 removed for safety.
+const STRONG_SCALER_THRESHOLD = myBuffedstat(myPrimestat()) - 50;
```
Comment (horrible-little-slime) [inline packages/garbo/src/lib.ts:1152]: This is unrelated to this PR; remove it, and we can discuss it at another time

Comment (Rinn) [inline packages/garbo/src/lib.ts]:
```suggestion
const STRONG_SCALER_THRESHOLD = 1_000;
```

**Review by Rinn — APPROVED** [review]
